test/libsolidity/syntaxTests/operators/userDefined/using_for_empty_operator_name.sol

```diff
@@ -0,0 +1,7 @@
+using {f as ''} for uint;
+using {f as ""} for int;
+using {f as hex""} for address;
+// ----
+// ParserError 4403: (12-14): Not a user-definable operator: . Only the following operators can be user-defined: |, &, ^, ~, +, -, *, /, %, ==, !=, <, >, <=, >=
```

Comment:
This message looks as if it was malformed. Maybe it would be better to shorten it to this when we hit this case?
```suggestion
// ParserError 4403: (12-14): Not a user-definable operator. Only the following operators can be user-defined: |, &, ^, ~, +, -, *, /, %, ==, !=, <, >, <=, >=
```

Reply:
But in typical cases, this will yield an error like `Not a user-definable operator:x`, which is a bit ugly as well. It would be much nicer to have a specific error for this particular case, but then @ekpyron might have a stroke, so I guess I'm more for leaving it as is.

Reply:
I mean, honestly, if we want to be nitpicky about this, the parser is not really the place for erroring out on these anyways - what really should happen is to accept anything that's an operator here and just make this a generic ``Expected: operator`` as far as the parser is concerned - and then syntax check the operator in these things to restrict to supported operators... but none of this is worth spending time on, so just fix the ICE and be fine with the messages :-).

Reply:
100% agree.